File: solvers.julia/relpose_7p_fuv_angle/inidata_num_relpose_7p_fuv_angle.jl
function inidata_num_relpose_7p_fuv_angle()

    data = Vector{Matrix{Float64}}()
    push!(data,randn(3,2))
    data[1] = [data[1] data[1][:,1]*randn()+data[1][:,2]*randn()]
    push!(data,randn(1,1))
    return data

end

File: solvers.matlab/relpose_7p_fuv_angle/inidata_num_relpose_7p_fuv_angle.m
function data = inidata_num_relpose_7p_fuv_angle()

    data = cell([1,2]);
    data{1} = randn(3,2);
    data{1} = [data{1} data{1}(:,1)*randn+data{1}(:,2)*randn];
    data{2} = randn;

end